fairlearn/metrics/_fairness_metrics.py

```diff
@@ -197,3 +197,95 @@ def _get_eo_frame(y_true, y_pred, sensitive_features, sample_weight) -> MetricFr
         sample_params=sp,
     )
     return eo
+
+
+def equal_opportunity_difference(
```

Comment:
Well, I tried and importing `from fairlearn.metrics import true_positive_rate_difference` causes a circular import. Also, I think the current approach follows the same structure as the other fairness metrics in this file. E.g. `demographic_parity_difference` is using `MetricFrame` with `selection_rate` and then `difference` is being called instead of using a derived metric `selection_rate_difference` directly. Maybe it's because of that reason as well. I think the current approach is the closest to the other metrics implemented there.

Reply:
Oh, I suppose you're right. At least it's not a lot of code.